migrations_lockfile.txt

```diff
@@ -10,7 +10,7 @@ auth: 0008_alter_user_username_max_length
 contenttypes: 0002_remove_content_type_name
 jira_ac: 0001_initial
 nodestore: 0001_initial
-sentry: 0049_auto_20200304_0254
+sentry: 0050_fix_audit_pickled_teams
```

Comment:
This doesn't match `0050_fix_auditlog_pickled_data`?

Reply:
It's actually fine, the main thing that matters is that my pr changes this value, so that we generate a merge conflict if someone else also changes this value. I'll fix it regardless, since it looks confusing.